lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -66,6 +69,12 @@
     help='key for the table of new parameters',
 )
 # Optional arguments
+parser.add_argument(
+    '--catB-calibration-file',
+    dest='catB_calibration_file',
```

Comment:
dest is not needed here, since the same name will be generated automatically from the option name